capabilities.json

```diff
@@ -42,35 +42,76 @@
   ],
   "objects": {
     "group": {
-      "displayName": "Group",
-      "displayNameKey": "Visual_Group",
       "properties": {
         "showSelected": {
-          "displayName": "Show category label",
-          "displayNameKey": "Visual_ShowCategoryLabel",
```

Comment:
Do not remove displayNameKey strings

Reply:
See https://github.com/microsoft/powerbi-visuals-sunburst/pull/95#discussion_r1283043020. We do not need the keys here.